### PR TITLE
Overview docs

### PR DESCRIPTION
**Pull request by shannonxtreme**
Finishing up the Overview section

 - Added acronyms to the Glossary
 - Added Getting Started with links

### CODE REVIEW

File: Docs/source/Overview/hardware-overview.rst [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+There are two temperature sensors for each battery module with a module minion board for each of the four rows.
+Each board measures up to 12 battery modules and 16 temperature sensors, but is configured for eight modules and 
+16 sensors by default.
+
+The Amperes minion board is connected to the main power cable that powers the electrical loop using the :term:`Hall effect sensor`. 
+
+Leader board
+------------
+
+The Leader board uses the STM32F413 microcontroller. The board requires nine connectors:
```
Comment (shannonxtreme) [inline Docs/source/Overview/hardware-overview.rst:34]: Says nine connectors but only 8 total in the list.

File: Docs/source/Overview/hardware-overview.rst [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+* One 2-pin power connector (+12V, PWRGND)
+* One 2-pin error light connector (+, -)
+* One 4-pin contactor connector (+, -, aux1, aux2)
+* One 4-pin current board connector (+5V, GND, Analog signal 1, Analog signal 2)
+* One 4-pin CAN connector (isolated1 +5V, isolated1 GND, CAN high, CAN low)
+* One 2-pin Voltage connector (IP, IM)
+* Two 2x4-pin fan connectors (+12V, PWRGND)
```
Comment (shannonxtreme) [inline Docs/source/Overview/hardware-overview.rst:42]: What is 2x4-pin? Only two listed in the parentheses.

File: Docs/source/Overview/hardware-overview.rst [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+The STM32F413 requires bypass capacitors and uses an 8MHz clock. To program the STM32F413, pins are connected to a 1x4 header to use the :abbr:`SWD (Serial Wire Debug)` programming protocol. 
+
+The reset button allows us to avoid powering the BPS off to reset it.
+
+.. note::
+    You can configure the system to reset every time you use the Keil IDE to program it. 
+
+For more information, see the STM32F413 datasheet.
```
Comment (shannonxtreme) [inline Docs/source/Overview/hardware-overview.rst:51]: Link to datasheet?

File: Docs/source/Overview/hardware-overview.rst (hunk 2 above)
Comment (manthanand) [reply Docs/source/Overview/hardware-overview.rst:42]: It is 12V and PWRGND for each of the 4 fans.

File: Docs/source/Overview/hardware-overview.rst (hunk 3 above)
Comment (manthanand) [reply Docs/source/Overview/hardware-overview.rst:51]: https://www.st.com/resource/en/reference_manual/dm00305666-stm32f413-423-advanced-arm-based-32-bit-mcus-stmicroelectronics.pdf

File: Docs/source/Overview/hardware-overview.rst (hunk 2 above)
Comment (ClarkPoon) [reply Docs/source/Overview/hardware-overview.rst:42]: It has 8 pins, and they are arranged in a 2x4 grid

Comment (SijWoo) [reply Docs/source/Overview/hardware-overview.rst:42]:
```suggestion
* One 2-pin power connector (+12V, PWRGND)
* One 2-pin error light connector (+12V, PWRDND)
* One 4-pin contactor connector (+12V, PWRGND, aux1, aux2)
* One 4-pin Amperes board connector (+12V, PWRGND, IP, IM)
* One 4-pin CAN connector (isolated +5V, isolated GND, CAN high, CAN low)
* One 2-pin Minion connector (IP, IM)
* One 2x4-pin fan connectors (+12V x4, PWRGND x4)
```
Here are the updated connections

Comment (SijWoo) [inline Docs/source/Overview/hardware-overview.rst:42]:
```suggestion
* One 2-pin power connector (+12V, PWRGND)
* One 2-pin error light connector (+12V, PWRDND)
* One 4-pin contactor connector (+12V, PWRGND, aux1, aux2)
* One 4-pin Amperes board connector (+12V, PWRGND, IP, IM)
* One 4-pin CAN connector (isolated +5V, isolated GND, CAN high, CAN low)
* One 2-pin Minion connector (IP, IM)
* One 2x4-pin fan connectors (+12V x4, PWRGND x4)
```
Here are the updated connections

File: Docs/source/glossary.rst [hunk 4]
```diff
@@ -5,4 +5,34 @@ Glossary
 .. glossary::
```
Comment (ClarkPoon) [inline Docs/source/glossary.rst:6]: We have a more extensive glossary that I pasted into the google doc underneath this one.

Comment (shannonxtreme) [reply Docs/source/glossary.rst:6]: Go for it! The glossary should ideally not be used for abbreviations, which should be in-line where it makes sense, but I think with the complexity of this documentation having a page that defines all this is not a bad idea. 

I don't think it's a great idea to have this in the Glossary, so I'd separate it into Glossary and a separate page for Abbreviations

File: Docs/source/Overview/hardware-overview.rst (hunk 2 above)
Comment (shannonxtreme) [reply Docs/source/Overview/hardware-overview.rst:42]: Thanks. What are IP and IM, and also is it total seven connectors then?

File: Docs/source/Overview/getting-started.rst [hunk 5]
```diff
@@ -4,8 +4,7 @@ Getting Started
 
 To work with the BPS, download the following software: 
 
-* `Kicad PCB Editor <https://kicad-pcb.org/>`_ - View :abbr:`SCH (pending)` and PCBs
+* `Kicad PCB Editor <https://kicad-pcb.org/>`_ - View :abbr:`SCH (Circuit Schematic File)` and PCBs
```
Comment (shannonxtreme) [inline Docs/source/Overview/getting-started.rst]: Is this abbr definition needed or are even beginners expected to know what an sch file is?

File: Docs/source/Overview/hardware-overview.rst [hunk 6]
```diff
@@ -31,22 +31,22 @@ The Amperes minion board is connected to the main power cable that powers the el
 Leader board
 ------------
 
-The Leader board uses the STM32F413 microcontroller. The board requires nine connectors:
+The Leader board uses the STM32F413 microcontroller. The board requires seven connectors:
```
Comment (shannonxtreme) [inline Docs/source/Overview/hardware-overview.rst]: Updated to seven connectors

File: Docs/source/HardwareOverview/boards-sensors.rst [hunk 7]
```diff
@@ -27,4 +27,42 @@ The reset button allows you to avoid powering the BPS off to reset it.
...
+.. figure:: ../_static/crystal-capacitors.png
+    :align: center
+
+    STM32 crystal layout
+
+.. _power-connector:
+
+Power connector
+^^^^^^^^^^^^^^^
+The Leader board gets a 12V power supply. The voltage must be dropped to power the MCU and components. The total voltage required for all the components is +3.3V and +5V.
```
Comment (shannonxtreme) [inline Docs/source/HardwareOverview/boards-sensors.rst:58]: Why are there two total voltage values?

Comment (ClarkPoon) [reply Docs/source/HardwareOverview/boards-sensors.rst:58]: We have a 12V power line that the BPS gets its power from. Some of the electronic components require 3.3V for power and some require 5V. The voltage needed to power each component is specified in their datasheet. We get 3.3V and 5V by converting 12V into 3.3V and 5V.

File: Docs/source/Overview/getting-started.rst (hunk 5 above)
Comment (ClarkPoon) [reply Docs/source/Overview/getting-started.rst]: They don't need to know the abbreviation. I think "Circuit Schematic" should be sufficient.

**Pull request merged**